package.json

```diff
@@ -8,29 +8,7 @@
     "appId": "io.lbry.LBRY",
     "mac": {
       "category": "public.app-category.utilities",
-      "target": "dmg"
-    },
-    "dmg": {
-      "iconSize": 128,
```

Comment:
As far as I can tell, without using xcode there is no way to use an icon for the .pkg